packages/odf/constants/kms.ts

```diff
@@ -94,8 +99,49 @@ export const HpcsEmptyState: HpcsConfig = Object.seal({
   hasHandled: true,
 });
 
-export const KMSEmptyState: KMSConfig = Object.seal({
+export const ThalesEmptyState: ThalesConfig = Object.seal({
```

Comment:
```suggestion
const ThalesEmptyState: ThalesConfig = Object.seal({
```
Let's make `ProviderStateMap` the only way to access these empty states outside of the file.

Reply:
ack...